src/Features/LanguageServer/Microsoft.CodeAnalysis.LanguageServer.Xaml/XamlLspWorkspaceManagerFactory.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Composition;
+using Microsoft.CodeAnalysis.Host.Mef;
+
+namespace Microsoft.CodeAnalysis.LanguageServer.Xaml;
+
+[ExportLspServiceFactory(typeof(LspWorkspaceManager), StringConstants.XamlLspLanguagesContract), Shared]
+internal class XamlLspWorkspaceManagerFactory : LspWorkspaceManagerFactory
```

Comment:
dont' like thsi either.  our concrete types should be considered sealed and should not be derived from.

Reply:
We do have this today already for the XAML only LSP server.  However - we're supporting XAML requests in the main Roslyn server, so there should be no need to export these anymore.

Otherwise you'll have different solution snapshots between XAML and C#

Reply:
You're right, I just verified it's not needed at all.

Reply:
Gone